projects/landing-page/src/App.tsx

```diff
@@ -267,7 +266,7 @@ const App: React.FunctionComponent = () => {
             </SectionText>
             <CardProject
               imageProps={{
-                path: Extension,
+                path: Extension as string,
```

Comment:
Why are these `as string` needed all of a sudden?

Reply:
These are needed in order to avoid the `/* eslint-disable @typescript-eslint/no-unsafe-assignment */` disabling at the beginning of the file